src/index.js

```diff
@@ -0,0 +1,96 @@
+import './style.css';
+
+document.querySelector('.footer-text').innerHTML = `&copy; ${new Date().getFullYear()} Henry-Kc, built with 💕 from me`;
+
+let toBeDeleted = [];
+const generateToDoRows = (text, index) => {
+  const div = document.createElement('div');
+  div.classList.add('to-do-row', 'custom-row');
+
+  const div2 = document.createElement('div');
+  div2.classList.add('two');
+
+  const input = document.createElement('input');
+  input.classList.add('checkbox');
+  input.type = 'checkbox';
+  input.addEventListener('click', () => {
+    const nextToCheckBox = input.nextElementSibling;
+    if (input.checked) {
+      nextToCheckBox.style.textDecorationLine = 'line-through';
+      nextToCheckBox.style.color = '#ccc';
+      nextToCheckBox.style.textDecorationThickness = '2px';
+      toBeDeleted.push(index);
+    } else {
+      nextToCheckBox.style.textDecorationLine = 'none';
+      nextToCheckBox.style.color = 'rgb(150, 145, 145)';
+      toBeDeleted.splice(toBeDeleted.indexOf(index), 1);
+    }
+  });
+  div2.appendChild(input);
+
+  const input2 = document.createElement('input');
```

Comment:
- [ ] Your _js_ module does not include `array object` of `tasks`. Kindly ensure that it has an array of some simple to do tasks (array of objects) with each object containing three keys - `description [string]`, `completed [bool]`, `index: [number]`. 
- [ ] Also, please ensure that on _page load_, the task array is rendered **dynamically** to the DOM. The tasks should appear in order of the `index` values for each task

Reply:
That has been done. Thanks @rloterh:pray: